### DISCUSSION TITLE
Method percentile_approx always gives nan for one of the quantiles

### DISCUSSION THREAD

**Discussion by AlexeyKonoshenkov** [vaexio/vaex#1517] [Q&A] [answered]
Hello,
I faced with the problem of getting nan for percentile_approx.
E.g. I'm creating a pandas dataframe filled with random numbers:

```
pand = pd.DataFrame(np.random.randn(300, 4), columns=list('ABCD'))
vae = vx.from_pandas(dr)
```

```
pand.A.quantile([0.5])
0.5    0.079715
```

```
vae.percentile_approx("A", percentage=[50])
vae.percentile_approx("A", percentage=50)

array([nan])
nan
```

If I set several quantiles:
```
pand.A.quantile([0.1, 0.5, 0.9])

0.1   -1.285145
0.5    0.079715
0.9    1.314101
Name: A, dtype: float64
```

```
vae.percentile_approx("A", percentage=[10,50,90])

array([        nan, -0.03551129,  1.2551046 ])
```
So in vaex the first percentile is always nan except the situation when the non-zero persentile is zero:
```
vae.percentile_approx("A", percentage=[0,10,20,50,90])

array([-3.01567987,         nan, -0.82706092, -0.03551129,  1.2551046 ])
```

Vaex versions are:
```
 'vaex': '4.3.0',
 'vaex-core': '4.3.0.post1',
 'vaex-viz': '0.5.0',
 'vaex-hdf5': '0.8.0',
 'vaex-server': '0.5.0',
 'vaex-astro': '0.8.2',
 'vaex-jupyter': '0.6.0',
 'vaex-ml': '0.12.0'
```

Seems the setting the needed quantile into the second position is the only solution to this?

**Comment by JovanVeljanoski** [comment, marked as answer]
Hi, There have been number of issues on this in the past. This issue is due to numpy problem.. nothing we can do about it. 
There was a "magic" version, i i think 1.19.5 or something around that, which worked.

Yeah I checked on linux (ubuntu) with numpy 1.19.5 your example works.

<img width="1160" alt="image" src="https://user-images.githubusercontent.com/18574951/129461380-046018be-e295-4189-8cc1-baa0e115b23a.png">

**Reply by AlexeyKonoshenkov** [reply]
Thank you very much! I'll try it with mentioned versions of numpy and hope for the magicality of next versions.
